vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -57,10 +57,10 @@ type AdvancedOptions =
     { IsBlockStructureEnabled: bool 
       IsOutliningEnabled: bool }
 
-[<Export(typeof<ISettings>)>]
```

Comment:
The reason this was exported and initialized independently was to handle the following scenario:
The user opens Visual Studio and without opening any solution goes directly to Tools -> Options -> Text Editor -> F#
FSharpPackage.Initialize() was not called in such case. I'm not sure how would async package behave. It is something we need to ensure still works.

Also, I regret not commenting this properly.